pdns/dnsdistdist/html/index.html

```diff
@@ -1,50 +1,47 @@
 <!DOCTYPE html>
-<head>                                                                  
+<head>
   <meta charset="utf-8" />
-  <script src="js/jquery-1.8.3.min.js"></script>        
-  <script src="js/d3.v3.js"></script>        
-  <script src="js/rickshaw.min.js"></script>        
-  <script src="js/purl.js"></script>
-  <script src="js/moment.min.js"></script>
+  <style>
+    #qpschart {
+    position: relative;
+    left: 40px;
+    }
+    #qpsy_axis {
+    position: absolute;
+    top: 0;
+    bottom: 0;
+    width: 40px;
+    }
+
+    #cpuchart {
+    position: relative;
+    left: 40px;
+    }
+    #cpuy_axis {
+    position: absolute;
+    top: 0;
+    bottom: 0;
+    width: 40px;
+    }
+
+    table.border, table.border>tbody>tr>td {
+    border: 1px solid black;
+    border-collapse: collapse;
+    }
+  </style>
```

Comment:
Why? I would argue to put everything in the html file, it gives no benefits to load seperate files
Anyway in this commit I just moved them from body to head